src/Agent/NewRelic/Profiler/Logging/Logger.h

```diff
@@ -127,15 +158,15 @@ namespace NewRelic {
                 }
 
                 using stream_char_t = typename _Log::char_type;
-                if (level >= log.GetLevel())
+                if (log.GetInitialized() && log.GetEnabled() && (level >= log.GetLevel()))
                 {
                     //each thread will have these on the stack...
                     std::tm  tstruct;
                     std::time_t now;
                     (void)time(&now);
                     (void)gmtime_s(&tstruct, &now);
                     auto levelstr = nrlog::GetLevelString(level);
-                    auto& strm = log.ostr();
+                    std::wostream& strm = log.GetConsoleLogging() ? std::wcout : log.ostr();
```

Comment:
Will `wcout` work across platforms for us? I think there are cases where the `L` prefix is used strings that we log, no prefix is used for the strings, on Linux some of the strings will be prefixed with `u`, and then we also log strings of the type xstring_t which are different between windows and linux.

Reply:
That's a good question, I'll have to look into it.

Reply:
I think we're fine. The stream insertion operator `<<` should already be handling the conversion so that everything is the same string type, and Alex and I just ran a quick test and all the output was correct. We can also do some additional testing at the end of the milestone.